iOS/LocalPackages/DataBrokerProtection-iOS/Sources/DataBrokerProtection-iOS/DataBrokerProtectionIOSManager.swift

```diff
@@ -105,41 +105,45 @@ public class DataBrokerProtectionIOSManagerProvider {
         let database = DataBrokerProtectionDatabase(fakeBrokerFlag: fakeBroker, pixelHandler: sharedPixelsHandler, vault: vault, localBrokerService: localBrokerService)
 
         let operationQueue = OperationQueue()
-        let operationsBuilder = DefaultDataBrokerOperationsCreator()
+        let jobProvider = BrokerProfileJobProvider()
         let mismatchCalculator = DefaultMismatchCalculator(database: database,
                                                            pixelHandler: sharedPixelsHandler)
 
-        let queueManager =  DefaultDataBrokerProtectionQueueManager(operationQueue: operationQueue,
-                                                                    operationsCreator: operationsBuilder,
-                                                                    mismatchCalculator: mismatchCalculator,
-                                                                    pixelHandler: sharedPixelsHandler)
+        let brokerUpdater = RemoteBrokerJSONService(featureFlagger: featureFlagger,
+                                                    settings: dbpSettings,
+                                                    vault: vault,
+                                                    authenticationManager: authenticationManager,
+                                                    pixelHandler: sharedPixelsHandler,
+                                                    localBrokerProvider: localBrokerService)
```

Comment:
I need to check why this is considered a new change, it was a change that came in from `main` as part of the many merge conflicts at the end of last week. I'll figure this out at the start of my day tomorrow and double check the rest of the PR.

Reply:
Might be worth getting Anh to have a glance too to see if things are as he expects

Reply:
Yeah agreed, I have a testing task assigned for the remote JSON broker delivery too, so I'm going to figure out what's up with this diff then run through those from this PR and see how things look. (And get Anh to check anyway)